commcare_export/cli.py

```diff
@@ -49,8 +44,8 @@ def main(argv):
     parser.add_argument('--project')
     parser.add_argument('--username')
     parser.add_argument('--password', help='Enter password, or if using apikey auth-mode, enter the api key.')
-    parser.add_argument('--auth-mode', default='digest', help='Use "session" based auth, "digest" auth, or'
-                                                              ' "apikey" auth (for two factor enabled domains).')
+    parser.add_argument('--auth-mode', default='digest', choices=['digest', 'apikey'],
+                        help='Use "digest" auth, or "apikey" auth (for two factor enabled domains).')
```

Comment:
What's the impetus behind this change?  Is it the case that session auth doesn't offer any advantage over digest in this context and so there's no point to support both?

Reply:
Yes, session auth was originally the default but was replaced with Digest some time ago.